packages/greeter/scripts/exec.js

```diff
@@ -144,14 +144,16 @@ const main = async () => {
   const ABI = ['function setGreeting(string _greeting)']
   const iface = new ethers.utils.Interface(ABI)
   const calldata = iface.encodeFunctionData('setGreeting', [newGreeting])
-
+  console.log(await l2Wallet.address)
```

Comment:
do we need this

Reply:
LOL! It was for testing! I forgot to remove it, thanks!